### PR TITLE
Use new hosted fonts

### PR DESCRIPTION
**Pull request by kyryl-bogach**
Avoid relying on google fonts (GDPR issues). 
Relevant [Slack thread](https://myworkout.slack.com/archives/G9J24A12M/p1666003150500209)

- Update `Muli` font with `Mulish` (it's just a rename, it is supossed to be the same font)
- Change default font weight from `300` to `400`.
- Introduce bold font weight: `700`
- Apply styles to only have `400` & `700` weights.

### CODE REVIEW

File: src/sass/fonts.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  font-weight: bold;
+  src: url(https://dist.myworkout.com/fonts/Mulish-Bold.ttf);
+}
+
+@font-face {
+  font-family: 'Baloo 2';
+  font-style: normal;
+  font-weight: bold;
+  src: url(https://dist.myworkout.com/fonts/Baloo2-Bold.ttf);
+}
```
Comment (kyryl-bogach) [inline src/sass/fonts.scss:20]: This can be out of the styles library maybe. It's only being used in the webapp 🤔

**Comment by hansnn** [comment]
Related issue: https://github.com/myworkout/webapp/issues/107

File: src/sass/fonts.scss (hunk 1 above)
Comment (kyryl-bogach) [reply src/sass/fonts.scss]: Fonts that are not used won't be imported, so this is fine 👍🏻

**Review by hansnn — APPROVED** [review]
👍🏻

**Review by grEvenX — APPROVED** [review]

**Pull request merged**